ledger/sandbox/src/main/scala/com/digitalasset/platform/configuration/LedgerConfigConfiguration.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.configuration
+
+import java.time.Duration
+
+import com.daml.ledger.participant.state.v1.{Configuration, TimeModel}
+
+/**
+  * Configuration surrounding parties and party allocation.
+  */
+case class LedgerConfigConfiguration(
```

Comment:
"Config" twice in the name is pretty confusing. How do you feel about `InitialLedgerConfiguration` or `DefaultLedgerConfiguration`?

Reply:
Or simply `LedgerConfiguration`?

Reply:
Eh… it's only the ledger configuration for a new ledger. If you change it, then it's a lie.

Reply:
Technically, it's the configuration of how the ledger API server writes the initial ledger configuration...
`InitialLedgerConfiguration` sounds better than `LedgerConfiguration` to me.